README.md

```diff
@@ -35,12 +35,24 @@ args:
 dirs:
   - var/data/tmp
   - var/log
+# OPTIONAL - A map of configurations of sub-processes to launch
+sub-processes:
+  SUB_PROCESS_NAME:
+    # another StaticLauncherConfig, cannot have it's own sub-processes
+    configType: executable
+    configVersion: 1
```

Comment:
remove to reflect update of when version is specified

Reply:
👍